### PR TITLE
Back up at midday and again in the evening

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
DDF wants two daily backups to be made; one around lunch and another in the evening. 
This PR changes the backup scheduling from running in the late evening to running at 12:00 and again at 19:00.

#### Should this be tested by the reviewer and how?
I don't see any good path to test this outside of the production envs. We should however have a look at the backup being made the day this PR is merged in. 

#### Any specific requests for how the PR should be reviewed?
Ensure that the times entered at the hours place is 12 and 19.

#### What are the relevant tickets?
https://reload.atlassian.net/browse/DDFDRIFT-35

### CODE REVIEW

File: infrastructure/dpladm/env-repo-template/standard/.lagoon.yml [hunk 1]
```diff
@@ -82,4 +82,4 @@ backup-retention:
     hourly: 0
 # https://docs.lagoon.sh/lagoon/using-lagoon-the-basics/lagoon-yml#backup-schedule
 backup-schedule:
-  production: M H(22-2) * * *
+  production: M 12,19 * * *
```
Comment (hypesystem) [inline infrastructure/dpladm/env-repo-template/standard/.lagoon.yml:85]: May be worth considering keeping the jitter (c.f. [what H does in cron syntax](https://stackoverflow.com/a/47302727/1080564)) here?

**Review by spaceo — COMMENTED** [review]
What about the daily rentention rate?
Should it be doubled now the backup frequency has been doubled?

File: infrastructure/dpladm/env-repo-template/standard/.lagoon.yml (hunk 1 above)
Comment (ITViking) [reply infrastructure/dpladm/env-repo-template/standard/.lagoon.yml:85]: yes, might be a good idea aswell. It does jitter on the minute though

**Comment by ITViking** [comment]
We actually talked about that as well. We should probably double it, to give ourselves enough time :)

**Comment by ITViking** [comment]
@spaceo i made the requested changes

**Review by hypesystem — APPROVED** [review]
Looks good to me

Would like @spaceo's final approval too, though, just because this is new stuff for us to be touching :smile:

**Review by spaceo — APPROVED** [review]
👍 LGTM

**Pull request merged**